airbyte-webapp/package.json

```diff
@@ -81,6 +81,7 @@
     "yup": "^0.32.11"
   },
   "devDependencies": {
+    "@airbyte/eslint-plugin": "file:./packages/eslint-plugin",
```

Comment:
:information_source: Add a local eslint plugin, so we can easier write custom linting rules in the future.